README.md

```diff
@@ -67,8 +67,7 @@ If there are changes made in the Original repository, you can sync the fork to k
 
 ## Requirements
 * [ArcGIS Runtime SDK for .NET 100.9 or higher](https://developers.arcgis.com/net/latest/)
-* [XCode 11 or higher](https://developer.apple.com/xcode/downloads/)
-    * App supports a minimum deployment target of iOS 12 but requires the iOS 13 SDK to build. Note, because ArcGIS Runtime uses [Metal](https://developer.apple.com/metal/), if you want to simulate the app using an iOS simulator you must be running, at minimum, macOS Catalina and iOS 13.
+* [XCode 12 or higher](https://developer.apple.com/xcode/downloads/) and the iOS 13 SDK.
```

Comment:
I believe Xcode 12 is not yet a requirement for using the iOS SDK. Has the .Net team made this a requirement of the Xamarin.iOS SDK?

```suggestion
* [Xcode 12 or higher](https://developer.apple.com/xcode/downloads/) and the iOS 13 SDK.
```

Reply:
It looks like we don't document a supported version, so probably best not to mention it. I know practice internally is to always use the latest Xcode, because typically Xamarin works best when everything is on latest.

Reply:
I changed it to : `[Xcode](https://developer.apple.com/xcode/downloads/) and the iOS 13 SDK. The latest version of Xcode is preferred`